lib/pentagon/default/vpc/Makefile

```diff
@@ -1,15 +1,21 @@
 .PHONY: all plan apply destroy provision ssh
```

Comment:
Bucket should be added as a `PHONY` target to avoid any file shenanigans by `make`.